Classes/ViewHelpers/Override/TranslateViewHelper.php

```diff
@@ -143,7 +143,8 @@ protected static function translate($id, $extensionName, $arguments, $languageKe
                 $languageKey = 'default';
             }
         } elseif (self::$extensionConfiguration->getCrowdinIdentifier()) {
-            if (str_contains($id, 'EXT:'.self::$extensionConfiguration->getExtensionKey())) {
+            if (str_contains($id, 'EXT:'.self::$extensionConfiguration->getExtensionKey())
```

Comment:
we should probably go over each file and apply (current) code style...

Reply:
yeah better use a csfixer